server/package.json

```diff
@@ -10,42 +10,41 @@
   "prisma": {
     "version": "d7866d1a109b061d955254b078e58be2615d4e31"
   },
+  "bin": {
+    "prisma-language-server-test": "./out/bin.js"
+  },
   "repository": {
     "type": "github",
-    "url": "https://github.com/prisma/vscode"
+    "url": "git+https://github.com/prisma/vscode.git"
   },
+  "main": "./out/index.js",
   "dependencies": {
     "@prisma/fetch-engine": "2.0.0-alpha.1187",
     "@prisma/get-platform": "2.0.0-alpha.1187",
     "@prisma/sdk": "2.0.0-alpha.1187",
     "pkg-dir": "4.2.0",
+    "prismafile": "1.0.3",
```

Comment:
What is the long term strategy with this dependency?

Reply:
The plan is to not use this dependency at all in future. Instead, a part of the Rust parser will probably be used.